openslides/config/serializers.py

```diff
@@ -0,0 +1,9 @@
+from openslides.utils.rest_api import serializers, VariableField
+
+
+class ConfigSerializer(serializers.Serializer):
```

Comment:
I would like to use angular-data to handle the config values. For that, the config have to be in the format like any other model. So I need a list of dicts and not a dict it self. Also, I need the possibility to get inject single config values that were updated and pushed via autoupdate. For that, tornado need to call a get request on a single config value.

I don't see the problem, that we use the the same functions like in any other app here.